android/build.gradle

```diff
@@ -228,7 +228,14 @@ dependencies {
     //noinspection GradleDynamicVersion
     extractJNI("com.facebook.fbjni:fbjni:+")
 
-    def rnAAR = fileTree("${nodeModules}/react-native/android").matching({ it.include "**/**/*.aar" }).singleFile
+    def buildType = "debug"
+    tasks.all({ task ->
+      if (task.name == "buildCMakeRelease") {
+        buildType = "release"
+      }
+    })
+    def rnAarMatcher = "**/react-native/**/*${buildType}.aar"
```

Comment:
Not too knowledgeable on the specifics, but this will break RN 0.67.X, and possibly others.

```console
λ ls node_modules/reactnative/android/com/facebook/react/react-native/0.67.2/                                       
react-native-0.67.2.aar
```

There seems to be no `buildType` defined.

Reply:
This can be added: 

```console
    if (REACT_NATIVE_VERSION < 69) {
       rnAarMatcher = "**/**/*.aar"
     }
```

as done in this PR: https://github.com/mrousavy/react-native-mmkv/pull/414/files